scenarios/throughput_stress.go

```diff
@@ -22,16 +22,13 @@ import (
 const (
 	IterFlag    = "internal-iterations"
 	IterTimeout = "internal-iterations-timeout"
-	// IterResumeAt is the iteration from which to resume a previous run from. If set, it will skip the
-	// run initialization and start the next iteration starting from the given iteration.
-	IterResumeAt      = "internal-iterations-resume-at"
-	SkipSleepFlag     = "skip-sleep"
-	CANEventFlag      = "continue-as-new-after-event-count"
-	NexusEndpointFlag = "nexus-endpoint"
-	// WorkflowIDPrefix is the prefix for each run's workflow ID. Use it to ensure that the workflow IDs are unique.
-	WorkflowIDPrefix = "workflow-id-prefix"
```

Comment:
Removing this option as it was a stopgap solution to dealing with Omes failures.